### PR TITLE
Cyclops Mining Vessel

### PR DESCRIPTION
**Pull request by Eyeveri**
This creates a specific vessel for Hephaestus Industries. This is one of several that will be PR'd. Do not merge until the lore team says to.

### CODE REVIEW

**Comment by Eyeveri** [comment]
!wip

**Comment by Eyeveri** [comment]
!do not merge

**Comment by Eyeveri** [comment]
Reopening with a different PR. I did a lot that broke a lot and i honestly am not sure what it all was. So, to avoid that I am just taking the four main files and the uniform code, then making a new branch to add.

File: code/modules/clothing/under/jobs/security.dm [hunk 1]
```diff
@@ -73,6 +73,14 @@
 	icon_state = "epmc_alt"
 	item_state = "epmc_alt"
 
+/obj/item/clothing/under/rank/security/heph //Ghost Role Only.
+    name = "hephaestus security officer uniform"
+    desc = "A green-and-orange uniform worn by Security Officers of smaller Hephaestus Industries vessels."
+    icon = 'icons/clothing/under/uniforms/heph.dmi'
```
Comment (RustingWithYou) [inline code/modules/clothing/under/jobs/security.dm:79]:
```suggestion
    icon = 'icons/clothing/under/uniforms/heph_security.dmi'
```
should fix the unit test failure

File: code/modules/clothing/under/jobs/civilian.dm [hunk 2]
```diff
@@ -237,3 +237,20 @@
...
 	icon_state = "orion_machinist"
 	item_state = "orion_machinist"
+
+/obj/item/clothing/under/rank/security/heph
+    name = "hephaestus security officer uniform"
+    desc = "A green-and-orange uniform worn by Security Officers of smaller Hephaestus Industries vessels."
+    icon = 'icons/clothing/under/uniforms/heph.dmi'
+    icon_state = "heph_security"
+    item_state = "heph_security"
+    contained_sprite = TRUE
```
Comment (RustingWithYou) [inline code/modules/clothing/under/jobs/civilian.dm:247]:
```suggestion
```
already defined in security.dm

**Review by RustingWithYou — CHANGES_REQUESTED** [review]

**Pull request closed without merging**